src/scriptworker/version.py

```diff
@@ -40,9 +44,9 @@ def get_version_string(version):
     """
     version_len = len(version)
     if version_len == 3:
-        version_string = "%d.%d.%d" % version
+        version_string = "%d.%d.%d" % cast(ShortVerType, version)
     elif version_len == 4:
-        version_string = "%d.%d.%d%s" % version
+        version_string = "%d.%d.%d%s" % cast(LongVerType, version)
```

Comment:
this dance was needed because mypy didn't identify the Tuple Length variants between the len== piece here, and flagged both branches of the `Union`, specifically too-few for the ==3 side, and too-many for the ==4